packages/driver/src/cypress/error_messages.js

```diff
@@ -864,7 +864,9 @@ module.exports = {
       This was never documented nor supported.
 
       Please go through the public function: ${cmd('state', '...')}`,
-    retry_timed_out: 'Timed out retrying: ',
+    retry_timed_out ({ ms }) {
+      return `Timed out retrying ${ms}ms: `
```

Comment:
Reword and also replace `:` with `.`, since oftentimes if ends up with double colons in error message, which just doesn't read as great.

```suggestion
      return `Timed out retrying after ${ms}ms. `
```